{
	"schema_version": "1.6.0",
	"id": "RHSA-2024:6962",
	"modified": "2024-12-18T10:09:02Z",
	"published": "2024-10-01T05:07:30Z",
	"related": [
		"CVE-2024-4032",
		"CVE-2024-6923",
		"CVE-2024-8088"
	],
	"summary": "Red Hat Security Advisory: python3.11 security update",
	"severity": [
		{
			"type": "CVSS_V3",
			"score": "CVSS:3.1/AV:N/AC:H/PR:L/UI:N/S:U/C:H/I:H/A:N"
		}
	],
	"affected": [
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:8::appstream",
				"name": "python3.11",
				"purl": "pkg:rpm/redhat/python3.11"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.11.9-7.el8_10"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2024:6962.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:8::appstream",
				"name": "python3.11-debug",
				"purl": "pkg:rpm/redhat/python3.11-debug"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.11.9-7.el8_10"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2024:6962.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:8::appstream",
				"name": "python3.11-debuginfo",
				"purl": "pkg:rpm/redhat/python3.11-debuginfo"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.11.9-7.el8_10"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2024:6962.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:8::appstream",
				"name": "python3.11-debugsource",
				"purl": "pkg:rpm/redhat/python3.11-debugsource"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.11.9-7.el8_10"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2024:6962.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:8::appstream",
				"name": "python3.11-devel",
				"purl": "pkg:rpm/redhat/python3.11-devel"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.11.9-7.el8_10"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2024:6962.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:8::appstream",
				"name": "python3.11-idle",
				"purl": "pkg:rpm/redhat/python3.11-idle"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.11.9-7.el8_10"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2024:6962.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:8::appstream",
				"name": "python3.11-libs",
				"purl": "pkg:rpm/redhat/python3.11-libs"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.11.9-7.el8_10"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2024:6962.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:8::appstream",
				"name": "python3.11-rpm-macros",
				"purl": "pkg:rpm/redhat/python3.11-rpm-macros"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.11.9-7.el8_10"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2024:6962.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:8::appstream",
				"name": "python3.11-test",
				"purl": "pkg:rpm/redhat/python3.11-test"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.11.9-7.el8_10"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2024:6962.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:8::appstream",
				"name": "python3.11-tkinter",
				"purl": "pkg:rpm/redhat/python3.11-tkinter"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.11.9-7.el8_10"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2024:6962.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:8::crb",
				"name": "python3.11",
				"purl": "pkg:rpm/redhat/python3.11"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.11.9-7.el8_10"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2024:6962.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:8::crb",
				"name": "python3.11-debug",
				"purl": "pkg:rpm/redhat/python3.11-debug"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.11.9-7.el8_10"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2024:6962.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:8::crb",
				"name": "python3.11-debuginfo",
				"purl": "pkg:rpm/redhat/python3.11-debuginfo"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.11.9-7.el8_10"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2024:6962.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:8::crb",
				"name": "python3.11-debugsource",
				"purl": "pkg:rpm/redhat/python3.11-debugsource"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.11.9-7.el8_10"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2024:6962.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:8::crb",
				"name": "python3.11-devel",
				"purl": "pkg:rpm/redhat/python3.11-devel"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.11.9-7.el8_10"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2024:6962.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:8::crb",
				"name": "python3.11-idle",
				"purl": "pkg:rpm/redhat/python3.11-idle"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.11.9-7.el8_10"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2024:6962.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:8::crb",
				"name": "python3.11-libs",
				"purl": "pkg:rpm/redhat/python3.11-libs"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.11.9-7.el8_10"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2024:6962.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:8::crb",
				"name": "python3.11-rpm-macros",
				"purl": "pkg:rpm/redhat/python3.11-rpm-macros"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.11.9-7.el8_10"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2024:6962.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:8::crb",
				"name": "python3.11-test",
				"purl": "pkg:rpm/redhat/python3.11-test"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.11.9-7.el8_10"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2024:6962.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:8::crb",
				"name": "python3.11-tkinter",
				"purl": "pkg:rpm/redhat/python3.11-tkinter"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.11.9-7.el8_10"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2024:6962.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/errata/RHSA-2024:6962"
		},
		{
			"type": "ARTICLE",
			"url": "https://access.redhat.com/security/updates/classification/#moderate"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2292921"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2302255"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2307370"
		},
		{
			"type": "ADVISORY",
			"url": "https://security.access.redhat.com/data/csaf/v2/advisories/2024/rhsa-2024_6962.json"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2024-4032"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2024-4032"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2024-4032"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2024-6923"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2024-6923"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2024-6923"
		},
		{
			"type": "ARTICLE",
			"url": "https://github.com/python/cpython/issues/121650"
		},
		{
			"type": "ARTICLE",
			"url": "https://github.com/python/cpython/pull/122233"
		},
		{
			"type": "ARTICLE",
			"url": "https://mail.python.org/archives/list/security-announce@python.org/thread/QH3BUOE2DYQBWP7NAQ7UNHPPOELKISRW/"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2024-8088"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2024-8088"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2024-8088"
		},
		{
			"type": "ARTICLE",
			"url": "https://github.com/python/cpython/commit/795f2597a4be988e2bb19b69ff9958e981cb894e"
		},
		{
			"type": "ARTICLE",
			"url": "https://github.com/python/cpython/commit/8c7348939d8a3ecd79d630075f6be1b0c5b41f64"
		},
		{
			"type": "ARTICLE",
			"url": "https://github.com/python/cpython/commit/dcc5182f27c1500006a1ef78e10613bb45788dea"
		},
		{
			"type": "ARTICLE",
			"url": "https://github.com/python/cpython/issues/122905"
		},
		{
			"type": "ARTICLE",
			"url": "https://github.com/python/cpython/pull/122906"
		},
		{
			"type": "ARTICLE",
			"url": "https://mail.python.org/archives/list/security-announce@python.org/thread/GNFCKVI4TCATKQLALJ5SN4L4CSPSMILU/"
		}
	]
}
